apps/test/unit/lib/levelbuilder/lesson-editor/LessonEditorTest.js

```diff
@@ -376,4 +376,14 @@ describe('LessonEditor', () => {
 
     server.restore();
   });
+
+  it('should render "Add Rubric" button when hasRubric prop is false', () => {
+    const wrapper = createWrapper({hasRubric: false});
+    expect(wrapper.find('.btn.add-rubric').text()).to.contain('Add Rubric');
+  });
+
+  it('should render "Edit Rubric" button when hasRubric prop is true', () => {
+    const wrapper = createWrapper({hasRubric: true});
+    expect(wrapper.find('.btn.add-rubric').text()).to.contain('Edit Rubric');
```

Comment:
Wondering: Should I also check to see that the hrefs are accurate here?

Reply:
That sounds like a great idea! You should be able to do something like `wrapper.find('.btn.add-rubric').props().href` and check equality